src/sentry/models/scheduledeletion.py

```diff
@@ -40,14 +40,24 @@ class Meta:
 
     @classmethod
     def schedule(cls, instance, days=30, hours=0, data=None, actor=None):
-        record = cls.objects.create(
+        model_name = type(instance).__name__
+        record, created = cls.objects.create_or_update(
```

Comment:
I wonder if we should start using the build in django `update_or_create` to avoid the weirdness we have of sometimes returning an object and sometimes returning a count.

https://docs.djangoproject.com/en/2.2/ref/models/querysets/#update-or-create

I don't think our version does anything too exciting, it looks like it might just be a backport.

Reply:
Yes, we should start moving to the built-in one.